helm/fiaas-skipper/templates/rbac.yaml

```diff
@@ -0,0 +1,66 @@
+{{- if .Values.rbac.enabled }}
+kind: ClusterRole
+{{- if semverCompare "^1.8-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
Which versions does this match? I can't find any documentation on what happens when the version after the hyphen is `0`. Would it be better to just use `">=1.8"`?

Reply:
I've updated to use >=1.8